src/services/YotiService.ts

```diff
@@ -166,7 +166,11 @@ export class YotiService {
 			},
 		};
 
-		this.logger.info("REQUIRED DOCS", {"required docs": payloadJSON.required_documents})
+		if (selectedDocument.toUpperCase() === "UKPASSPORT" || selectedDocument.toUpperCase() === "NONUKPASSPORT") {
```

Comment:
dont think we allow expired NONUKPASSPORT

Reply:
Removed :)